app/src/main/java/com/hedvig/app/ui/compose/composables/buttons/PrimaryButtons.kt

```diff
@@ -11,15 +11,33 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.tooling.preview.Preview
-import com.airbnb.android.showkase.annotation.ShowkaseComposable
 import com.hedvig.app.R
 import com.hedvig.app.ui.compose.theme.HedvigTheme
 
 @Composable
-fun LargeContainedButton(
+fun PrimaryTextButton(
+    modifier: Modifier = Modifier,
+    text: String,
+    onClick: () -> Unit
+) {
+    PrimaryButton(
+        modifier = modifier,
+        content = {
+            Text(
+                text = text,
+                style = MaterialTheme.typography.button,
+                color = MaterialTheme.colors.onSecondary
```

Comment:
This is actually implicit when adding a Text-Composable as a child to Button, through the magic of LocalContentColor and LocalTextStyle 🙂